typescript/hello/src/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Context, StructuredReturn } from 'faas-js-runtime';
+
+/**
+ * Your HTTP handling function, invoked with each request. This is an example
+ * function that will just print (return) a Hello World.
+ *
+ * It can be invoked with `func invoke`
+ * It can be tested with `npm test`
+ *
+ * @param {Context} context a context object.
+ * @param {object} context.body the request body if any
+ * @param {object} context.query the query string deserialized as an object, if any
+ * @param {object} context.log logging object with methods for 'info', 'warn', 'error', etc.
+ * @param {object} context.headers the HTTP request headers
+ * @param {string} context.method the HTTP request method
+ * @param {string} context.httpVersion the HTTP protocol version
+ * See: https://github.com/knative/func/blob/main/docs/guides/nodejs.md#the-context-object
+ */
+const handle = async (context: Context, body: string): Promise<StructuredReturn> => {
+  // YOUR CODE HERE
+  context.log.info(`request received`);
+  context.log.info(body);
+  return {
+    body: 'Hello Typescript World!',
```

Comment:
Is this a JSON? If anything it should be in `''` so it is JSON String. Or it could be reworked to a JSON object.

Reply:
Or alternatively change content-type to `text/plain`.